requirements/base.txt

```diff
@@ -1,28 +0,0 @@
-django-bootstrap-form==3.2
```

Comment:
We need to keep these requirement files for the moment or it will break our current build.
Can we think of a 3 steps transition ? 

1. First add packaging
2. then change our current build to pip install
3. Finally remove the requirement files ?

Reply:
What about keeping the `requirements/base.txt` file with the following content:

```
-e .
```

And then remove it when the deployment procedure switched to `pip install fun-apps`?

Reply:
:+1:

Reply:
done